### DISCUSSION TITLE
Copy issue link on mobile app

### DISCUSSION THREAD

**Discussion by jcubic** [community/community#63904] [Mobile]
It seems there is no way to copy the link to the issue to add it to the comment or another issue. You need to remember the username, repo, and issue number or take a note of it, possibly with pen and paper.

**Comment by eliperkins** [comment]
Hey @jcubic , thanks for this feedback.

Writing down URLs seems really frustrating! I think we can make this better for you. Can I get some more information from ya to make sure I understand correctly?

- What platform are you on? (e.g., iOS or Android)
- What device are you using? (e.g., iPhone 14, Samsung Note 10+)
- What version of GitHub Mobile are you on? (e.g., v1.125.0, v1.101.2)
- When tapping on the Share button in top-right of the the navigation bar for an issue, are you able to see an option to "Copy"?

Thanks!

**Comment by jcubic** [comment]
Ahh, didn't noticed the share button. This should be first thing I've checked. But I was blind I didn't even see that button, I was searching in meatballs menu.You can close this. 16 sie 2023 16:14 Eli Perkins ***@***.***> napisał(a):
Hey @jcubic , thanks for this feedback.
Writing down URLs seems really frustrating! I think we can make this better for you. Can I get some more information from ya to make sure I understand correctly?
What platform are you on? (e.g., iOS or Android)What device are you using? (e.g., iPhone 14, Samsung Note 10+)What version of GitHub Mobile are you on? (e.g., v1.125.0, v1.101.2)When tapping on the Share button in top-right of the the navigation bar for an issue, are you able to see an option to "Copy"?
Thanks!
